test-harness/modules/generic_test_code.py

```diff
@@ -8,7 +7,52 @@
 log = logging.getLogger(__name__)
 
 
-def generic_unknown_user_is_forbidden_test(ar, auth_header, path):
+def ping_mesos_agent(ar,
```

Comment:
Very often when we play with networking, we issue `ping 8.8.8.8` to test network connectivity. `ping_mesos_agent` has the same role for testing cache - it indirectly tests if Mesos statecache was cached properly and given agent is reachable. It abstracts the act of:
* forming the right URL
* issuing the request
* checking if the response was sent by correct upstream

It will most probably have many more uses as the number of tests keeps growing, though.

I hope it helps :)